booster/library/Booster/Pattern/ApplyEquations.hs

```diff
@@ -232,8 +232,9 @@ fromCache tag t = eqState $ Map.lookup t <$> gets (select tag . (.cache))
 
 logWarn :: LoggerMIO m => Text -> m ()
 logWarn msg =
-    logMessage' $
-        msg <> " For more details, enable full context logging '--log-context \"*\"'"
+    withContext "warning" $ do
+        logMessage $
+            msg <> " For more details, enable full context logging '--log-context \"*\"'"
```

Comment:
If we don't display the warning by default, we don't have to provide this additional advice (which is a bit "whole-sale" anyway).
```suggestion
    withContext "warning" $ logmessage msg 
```
Or else, we leave `*>warning.` on by default (I think it would make sense) and let downstream dev.s silence it.

Reply:
I agree that we can remove the context advice. Regarding letting the downstream devs silencing the warnings: I don't think that would work. The best strategy I see is to disable by default and ask them to enable it if the the downstream tool is running with `--verbose`.